### PR TITLE
Increased versioncolumn coverage

### PR DESCRIPTION
**Pull request by DeepData-Diver**
### Submitter checklist
- [ ] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [ ] Ensure that the pull request title represents the desired changelog entry
- [ ] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -78,11 +78,23 @@
     </dependencies>
   </dependencyManagement>
   <dependencies>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.13.2</version>
+      <scope>test</scope>
+    </dependency>
```
Comment (MarkEWaite) [inline pom.xml:86]: This is provided by the parent pom.  It is not needed in this pom file.
```suggestion
```

**Review by MarkEWaite — APPROVED** [review]
Tests look very good.  One optional change that I'd like in this pull request so that we don't duplicate the junit test dependency

File: pom.xml [hunk 2]
```diff
@@ -78,11 +78,23 @@
...
     <dependency>
       <groupId>org.jenkins-ci.modules</groupId>
       <artifactId>instance-identity</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-core</artifactId>
+      <version>5.12.0</version>
```
Comment (MarkEWaite) [inline pom.xml:95]: Let's use the most recent release
```suggestion
      <version>5.14.2</version>
```

File: pom.xml (hunk 1 above)
Comment (DeepData-Diver) [reply pom.xml]: Removed.

File: pom.xml (hunk 2 above)
Comment (DeepData-Diver) [reply pom.xml]: Updated.

**Comment by MarkEWaite** [comment]
Thanks.  This looks great.  For future pull requests, please use a branch named something other than "master" so that I can use standard "gh" and "git" commands to interact with your pull request.

File: src/test/java/hudson/plugin/versioncolumn/VersionMonitorTest.java [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+        when(computer.getChannel()).thenReturn(channel);
+        when(channel.call(ArgumentMatchers.<MasterToSlaveCallable<String, IOException>>any()))
+                .thenReturn(differentVersion);
+        when(computer.isOffline()).thenReturn(false);
+
+        String result = descriptor.monitor(computer);
+
+        assertEquals(differentVersion, result);
+        verify(computer, never()).setTemporarilyOffline(anyBoolean(), any());
+    }
```
Comment (basil) [inline src/test/java/hudson/plugin/versioncolumn/VersionMonitorTest.java:131]: These newly-added tests are failing on recent cores as of jenkinsci/jenkins#9855.

**Pull request merged**